specifyweb/frontend/js_src/lib/components/WebLinks/index.tsx

```diff
@@ -154,6 +155,7 @@ export function WebLinkField({
             }}
           >
             {image}
+            <div className="text-transform-none">{localized(url ?? '')}</div>
```

Comment:
this might be equivalent, and simpler:

```suggestion
            <div className="text-transform-none">{url}</div>
```

Reply:
fyi: LocalizedString is enforced for direct children of Button, but not for indirect, which is why you don't need to call it here anymore